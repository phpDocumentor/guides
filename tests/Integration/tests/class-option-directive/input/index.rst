
=========
Some File
=========

.. note::
    :class: highlights

    A Note with a class


.. class:: class1

.. note::
    :class: class2

    A Note with a class

.. code-block:: php
    :class: classInOption

    <?php

    echo 'Hi';

.. class:: classInDirective

.. code-block:: php

    <?php

    echo 'Hi';
