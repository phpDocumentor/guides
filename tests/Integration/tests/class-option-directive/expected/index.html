<!DOCTYPE html>
<html lang="en">
<head>
    <title>Some File</title>

</head>
<body>
    <div class="section" id="some-file">
    <h1>Some File</h1>

            <div class="admonition note highlights">
        <p>A Note with a class</p>
</div>

            <div class="admonition note class2 class1">
        <p>A Note with a class</p>
</div>

            <pre class="classinoption"><code class="language-php">&lt;?php

echo &#039;Hi&#039;;</code></pre>
            <pre class="classindirective"><code class="language-php">&lt;?php

echo &#039;Hi&#039;;</code></pre>
    </div>

</body>
</html>
